libp2p/Host.cpp

```diff
@@ -757,6 +755,8 @@ void Host::run(boost::system::error_code const&)
     m_timer->async_wait(runcb);
 }
 
+// Called after thread has been started to perform additional class-specific state initialization (e.g. start
```

Comment:
Maybe move the comment inside the function. This looks like doxygen comment.

Reply:
I like doxygen comments 😃 We have other functions that are commented like that e.g. https://github.com/ethereum/aleth/blob/b92056768376d42b809e2a80afd3178b2dead29f/libp2p/Common.cpp#L48-L51

Also, Geth functions are commented like that e.g: https://github.com/ethereum/go-ethereum/blob/472c23a8015cd84b193d2d0efb4592a664de3c62/internal/ethapi/addrlock.go#L43-L48